users/tests/snapshots/snap_test_api.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+# snapshottest: v1 - https://goo.gl/zC4yUc
+from __future__ import unicode_literals
+
+from snapshottest import Snapshot
+
+snapshots = Snapshot()
+
+snapshots["test_guardians_query_normal_user 1"] = {
+    "data": {
+        "guardians": {
+            "edges": [
+                {
+                    "node": {
+                        "email": "angelahawkins@gmail.com",
+                        "firstName": "Jessica",
```

Comment:
I don't get it 😅 why would it match every time, if it just comes from `...Factory()`. doesn't factory generate new names and such on each run?

Reply:
like, if I pull this branch and run `pytest` I would get data that would not match with the snapshot? or how does it work 🙈

Reply:
The factories will generate the same values everywhere everytime (or at least they should :) ) because the random seed for those is set to a fixed value [here](https://github.com/City-of-Helsinki/kukkuu/pull/9/files#diff-5c3f086ee36c1e0e4e81d69639021286R13).

Reply:
aaaahh, now it all makes sense. 👍 thanks for clarigying!

Reply:
Actually, not sure how robust the thing is after all, as changing the connectionfields seems to have affected the snapshots :D 

I continue to evaluate the snapshot thing in this project and let's see how well it works for us. It sure seems handy if it works.